var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -40,6 +41,9 @@ def install(self, spec, prefix):
         if '+shared' in self.spec:
             configure_args.append("--enable-shared")
 
+        if not self.compiler.f77 and not self.compiler.fc:
+            configure_args.append('--enable-fortran=no')
```

Comment:
OK, so hypre has all kinds of problems. First, it creates `.so` files, not `.dylib` files. Basically, it is assuming that the world is ELF and everything else is "oh well". I'm just going to look into porting it to use the CMake build system that comes with it rather than fixing all of the ELF -> MachO changes in that `configure` script (without a `configure.ac`…).